test/units/parsing/test_splitter.py

```diff
@@ -94,17 +128,50 @@
     (u'One\n  Two\n    Three\n',
         [u'One\n ', u'Two\n   ', u'Three\n'],
         {u'_raw_params': u'One\n  Two\n    Three\n'}),
+    (u'\nOne\n  Two\n    Three\n',
+        [u'\n', u'One\n ', u'Two\n   ', u'Three\n'],
+        {u'_raw_params': u'\nOne\n  Two\n    Three\n'}),
 )
 
-SPLIT_ARGS = ((test[0], test[1]) for test in SPLIT_DATA)
-PARSE_KV = ((test[0], test[2]) for test in SPLIT_DATA)
+PARSE_KV_CHECK_RAW = (
+    (u'raw=yes', {u'_raw_params': u'raw=yes'}),
+    (u'creates=something', {u'creates': u'something'}),
+)
+
+PARSER_ERROR = (
+    '"',
+    "'",
```

Comment:
These two error cases, while not new behavior, are mentioned in a bug report: https://github.com/ansible/ansible/issues/28674